http_calls.py

```diff
@@ -7,6 +7,9 @@
 import traceback
 import urllib
 import urllib.request
+from abc import ABC, abstractmethod
+from hyper import HTTP20Connection, HTTP20Response
```

Comment:
How did you install this `hyper` module? fitnesse tests should not work actually

Reply:
pip install hyper

https://github.com/QSFT/DMP/pull/1658/files#diff-34cb84cee8ad4327ef44d201230a39c6R23

Reply:
Why you are referring something not public here? As far as I remember for PySlim tests we use this Dockerfile https://github.com/VolodymyrLavrenchuk/PySlim/blob/master/tests/waferslim/Dockerfile#L2